src/rpcgovernance.cpp

```diff
@@ -623,6 +623,18 @@ UniValue voteraw(const UniValue& params, bool fHelp)
 
 UniValue getgovernanceinfo(const UniValue& params, bool fHelp)
 {
+    int nLastSuperblock, nNextSuperblock;
+
+    // Get current block height
+    int nBlockHeight = (int)chainActive.Height();
+
+    // Get chain parameters
+    int nSuperblockStartBlock = Params().GetConsensus().nSuperblockStartBlock;
+    int nSuperblockCycle = Params().GetConsensus().nSuperblockCycle;
+
+    // Get first superblock
+    int nFirstSuperblock = nSuperblockStartBlock - (nSuperblockStartBlock % nSuperblockCycle) + nSuperblockCycle;
```

Comment:
There's one case where this gives the wrong result: when 
`nSuperblockStartBlock % nSuperblockCycle == 0`

which is unlikely but not impossible since nSuperblockStartBlock hasn't been determined for mainnet yet.